jupyter-on-gke/jupyter_config/config-selfauth.yaml

```diff
@@ -78,10 +78,20 @@ singleuser:
       ephemeral-storage: 10Gi
   nodeSelector:
     iam.gke.io/gke-metadata-server-enabled: "true"
+  extraEnv:
+    JUPYTER_ALLOW_INSECURE_WRITES: "true"
```

Comment:
Maybe add a comment explaining why this is needed/ what error you see without

Reply:
Added a comment on top explaining what this variable does, not sure why it doesn't show here.